english/Find_the_cow.md

```diff
@@ -0,0 +1,3 @@
+Can you [find the cow ?] (https://github.com/udacity/create-your-own-adventure/blob/master/english/Find_the_cow.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.